src/Block/Service/MenuBlockService.php

```diff
@@ -163,19 +164,12 @@ protected function getFormSettingsKeys()
             'required' => false,
         ];
 
-        $choices = $this->menuRegistry->getAliasNames();
-
-        // NEXT_MAJOR: remove SF 2.7+ BC
-        if (method_exists('Symfony\Component\Form\AbstractType', 'configureOptions')) {
-            $choices = array_flip($choices);
-
-            // choice_as_value options is not needed in SF 3.0+
-            if (method_exists('Symfony\Component\Form\FormTypeInterface', 'setDefaultOptions')) {
-                $choiceOptions['choices_as_values'] = true;
-            }
+        // choice_as_value options is not needed in SF 3.0+
```

Comment:
should we name it with `NEXT_MAJOR` ?

Reply:
Probably yes, to be honest when I search for code removal I usually start with method_exists class_exists and interface_exists, then 5.3 and finally NEXT_MAJOR